packages/svelte/src/compiler/phases/2-analyze/css/css-analyze.js

```diff
@@ -28,11 +28,19 @@ function is_global_block_selector(simple_selector) {
 	);
 }
 
+/**
+ *
+ * @param {Array<AST.CSS.Node>} path
+ */
+function is_in_global_block(path) {
+	return path.some((node) => node.type === 'Rule' && node.metadata.is_global_block);
+}
```

Comment:
Mmm i have the feeling this will not work here since `is_global_block` will be populated during the analyze walk. This might mean the it will only work with certain orders. Will explore this a bit

Reply:
Oh but this will be invoked only after we already set that to true.

Reply:
Should I change it to use `is_global_block_selector()` to not depend on analyze step (which set global flag to metadata)?

Reply:
Nope that checks if a node is the global selector, you want to check if you are inside...i think it should work